src/Toolkit/Toolkit.Maui/Compass/ICompass.cs

```diff
@@ -19,6 +19,7 @@ namespace Esri.ArcGISRuntime.Toolkit.Maui;
 /// <summary>
 /// Defines the public API for the MAUI compass control.
 /// </summary>
+[Obsolete("No longer in use")]

```

Comment:
The interface _is_ still in use, isn't it? The new view implements it.
Perhaps only CompassHandler should be deprecated &mdash; or even removed?

Reply:
it's not a new view - its the same view - I kept the interface on it for backwards compat reasons, but it isn't "in use"